packages/core/phase2/lib/generateOperations/resultClassHandlers/handleJudgementWithFinalResult.test.ts

```diff
@@ -73,67 +73,6 @@ describe("handleJudgementWithFinalResult", () => {
     expect(operations).toStrictEqual([{ code: PncOperation.DISPOSAL_UPDATED, data: undefined, status: "NotAttempted" }])
   })
 
-  it("should only return exception HO200124 when HO200124 and HO200108 conditions are met", () => {
```

Comment:
I changed it to when results are NOT on pnc because we want to test the logic of 
`  
if (!allResultsAlreadyOnPnc && hasUnmatchedPncOffences(aho, ccrId) && !offence.AddedByTheCourt) {
    return { operations: [], exceptions: [] }
  }
`

right? we want it to enter that block statement so allResultsAlreadyOnPnc must be false

Reply:
That's right. 👍